mock/service/service.go

```diff
@@ -27,9 +27,10 @@ var Manifest = map[string]string{
 }
 
 type Config struct {
-	DisableAttach bool
-	DriverName    string
-	AttachLimit   int64
+	DisableAttach         bool
+	DriverName            string
+	AttachLimit           int64
+	NodeExpansionRequired bool
```

Comment:
Do you also want ControllerExpansion to be configurable?  In case you want to do tests against a driver that doesn't support expansion?

Reply:
I think we could but can we do this as a follow up? I would also like to make online and offline configurable.